pkg/sbom/cyclonedx/marshal.go

```diff
@@ -234,8 +239,9 @@ func (e *Marshaler) rootComponent(r types.Report) (*core.Component, error) {
 		p, err := purl.NewPackageURL(purl.TypeOCI, r.Metadata, ftypes.Package{})
 		if err != nil {
 			return nil, xerrors.Errorf("failed to new package url for oci: %w", err)
-		} else if p.Type != "" {
-			root.PackageURL = &p
+		}
+		if p != nil {
```

Comment:
nit: Is there any reason to make it an independent if-statement rather than `else if`?

Reply:
I don't see point of using `else if`, because we return error when `err != nil`.
And independent `if` seems to me more readable.